### PR TITLE
updated all KAIO component docs with warning

### PR DESCRIPTION
**Pull request by gyfchong**
## Why
Temporarily while it is not common installation practice for Kaizen, I have added a notification so people won't ask us that many questions about why this isn't styled.

## What
Updated docs with a new `KaioNotification` telling people to read installation docs.

### CODE REVIEW

**Review by christian314159 — DISMISSED** [review]

File: packages/components/src/ButtonGroup/_docs/ButtonGroup.stories.tsx [hunk 1]
```diff
@@ -11,21 +12,34 @@ export default {
...
+ * Used to group buttons. Compatible with buttons wrapped by Tooltip. Currently only compatible with Filter Buttons.
+ */
+export const Playground: StoryFn<typeof ButtonGroup> = args => (
   <ButtonGroup {...args}>
     <FilterButtonBase>First</FilterButtonBase>
     <FilterButtonBase>Last</FilterButtonBase>
   </ButtonGroup>
 )
-DefaultStory.storyName = "Button Group"
+Playground.storyName = "Button Group"
```
Comment (HeartSquared) [inline packages/components/src/ButtonGroup/_docs/ButtonGroup.stories.tsx:42]:
```suggestion
```
Pretty sure `Playground` should remain as `Playground`

File: packages/components/src/Filter/_docs/Filter.stories.tsx [hunk 2]
```diff
@@ -36,7 +53,7 @@ export const DefaultStory: StoryFn<typeof Filter> = () => {
     </Filter>
   )
 }
-DefaultStory.storyName = "Filter"
+Playground.storyName = "Filter"
```
Comment (HeartSquared) [inline packages/components/src/Filter/_docs/Filter.stories.tsx:56]:
```suggestion
```

File: packages/components/src/Filter/_docs/Filter.stories.tsx [hunk 3]
```diff
@@ -14,15 +15,31 @@ export default {
...
+        "https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3082093959/Filters",
     },
   },
 } satisfies Meta<typeof Filter>
 
-export const DefaultStory: StoryFn<typeof Filter> = () => {
+/**
+ * These components are mainly used for constructing Filters (eg. FilterSelect):
+ *
+ * - `FilterButton`for the renderTrigger.
```
Comment (HeartSquared) [inline packages/components/src/Filter/_docs/Filter.stories.tsx:39]:
```suggestion
 * - `FilterButton` for the renderTrigger.
```

Comment (HeartSquared) [reply packages/components/src/Filter/_docs/Filter.stories.tsx:39]: Would be nice if we can link to the FilterButton docs :')

File: packages/components/src/FilterButton/_docs/filter-buttons.mdx [hunk 4]
```diff
@@ -1,13 +1,14 @@
-import { Canvas, Controls, DocsStory, Meta, Source } from "@storybook/blocks"
+import { Canvas, Controls, DocsStory, Meta, Source, Unstyled } from "@storybook/blocks"
```
Comment (HeartSquared) [inline packages/components/src/FilterButton/_docs/filter-buttons.mdx:1]:
```suggestion
import { Canvas, Controls, DocsStory, Meta, Source } from "@storybook/blocks"
```

File: packages/components/src/KaizenProvider/_docs/KaizenProvider.mdx [hunk 5]
```diff
@@ -29,18 +32,20 @@ import { KaizenProvider } from '@kaizen/components';
 
 `KaizenProvider` at this version is a direct replacement for `ThemeProvider`, and is safe to do a 1 to 1 replacement.
 
-<Highlight className="diff">{migrationToV1}</Highlight>
+
+<Highlight className="diff shadow-sm">{migrationToV1}</Highlight>
+
```
Comment (HeartSquared) [inline packages/components/src/KaizenProvider/_docs/KaizenProvider.mdx:37]: You are a strange man. You remove blank lines in some docs, yet add double in places like this? 😂

File: storybook/components/DocsContainer/ComponentDocsTemplate/ComponentDocsTemplate.tsx [hunk 6]
```diff
@@ -40,6 +41,9 @@ export const ComponentDocsTemplate = ({
       <Content>
         <Title context={context} />
         <Links context={context} />
+        <div className="!mt-8">
```
Comment (HeartSquared) [inline storybook/components/DocsContainer/ComponentDocsTemplate/ComponentDocsTemplate.tsx:44]: But if it's not in KAIO, wouldn't we get too much margin at the top?

File: storybook/components/DocsContainer/ComponentDocsTemplate/subcomponents/KaioInstallation/KaioInstallation.tsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { KaioNotification } from "./KaioNotification"
+
+export type TitleProps = {
```
Comment (HeartSquared) [inline storybook/components/DocsContainer/ComponentDocsTemplate/subcomponents/KaioInstallation/KaioInstallation.tsx:4]:
```suggestion
export type KaioInstallationProps = {
```

File: storybook/components/DocsContainer/ComponentDocsTemplate/ComponentDocsTemplate.tsx (hunk 6 above)
Comment (gyfchong) [reply storybook/components/DocsContainer/ComponentDocsTemplate/ComponentDocsTemplate.tsx:44]: Ah you're right, this should be in the notification

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**